Lib/fontbakery/profiles/shared_conditions.py

```diff
@@ -157,16 +157,45 @@ def glyph_metrics_stats(ttFont):
     """Returns a dict containing whether the font seems_monospaced,
     what's the maximum glyph width and what's the most common width.
 
-    For a font to be considered monospaced, at least 80% of
-    the ascii glyphs must have the same width."""
+    For a font to be considered monospaced, if at least 80% of ASCII
+    characters have glyphs, then at least 80% of must have the same width,
```

Comment:
"80% of must" (seems to be missing a word here)

Reply:
Fixed to "80% of those must".